src/irr/asset/bawformat/legacy/CBAWLegacy.h

```diff
@@ -137,6 +137,52 @@ using BAWFileV2 = BAWFileVn<2>;
 using BlobHeaderV2 = BlobHeaderVn<2>;
 
 
+#include "irr/irrpack.h"
+//! Simple struct of essential data of ICPUMeshBuffer that has to be exported
+struct IRR_FORCE_EBO MeshBufferBlobV2 : TypedBlob<MeshBufferBlobV2, ICPUMeshBuffer>, FixedSizeBlob<MeshBufferBlobV2, ICPUMeshBuffer>
+{
+    //! Constructor filling all members
+    explicit MeshBufferBlobV2(const ICPUMeshBuffer* _mb)
```

Comment:
this constructor will never be used again, it can go 
(you will never make a new legacy blob from object, only new blob from old blob)

Reply:
okay, to remove